src/styles/theming/sap_fiori_3.scss

```diff
@@ -283,4 +283,17 @@
   --fdProgress_Icon_Positive: '';
   --fdProgress_Icon_Negative: '';
   --fdProgress_Icon_Critical: '';
+
+  /** Illustrated Message */
+  --sapIllus_BrandColorPrimary: var(--sapContent_Illustrative_Color1);
+  --sapIllus_BrandColorSecondary: var(--sapContent_Illustrative_Color2);
+  --sapIllus_AccentColor: var(--sapContent_Illustrative_Color3);
+  --sapIllus_StrokeDetailColor: var(--sapContent_Illustrative_Color4);
+  --sapIllus_Layering1: var(--sapContent_Illustrative_Color5);
+  --sapIllus_Layering2: var(--sapContent_Illustrative_Color6);
+  --sapIllus_BackgroundColor: var(--sapContent_Illustrative_Color7);
+  --sapIllus_ObjectFillColor: var(--sapContent_Illustrative_Color8);
+  --sapIllus_NoColor: none;
+  --sapIllus_PatternShadow: url(#sapIllus_PatternShadow);
+  --sapIllus_PatternHighlight: url(#sapIllus_PatternHighlight);
```

Comment:
Question: why are these variables introduced as theming variables? The values are the same in all themes.

Reply:
Otherwise, we won't see colors in the different themes

![image](https://user-images.githubusercontent.com/20265336/142836268-9a99b7a5-246b-43e6-9914-554d8e54e71b.png)

Reply:
Even if they are added in the root?

Reply:
Yea, if styles are present in `sap_fiori_3.scss` and not present in any theme files colors are not shown.